drizzlepac/align.py

```diff
@@ -164,10 +155,11 @@ def check_and_get_data(input_list, **pars):
 
 # ------------------------------------------------------------------------------------------------------------
 
+
 def perform_align(input_list, archive=False, clobber=False, debug=False, update_hdr_wcs=False, result=None,
-              runfile=None, print_fit_parameters=True, print_git_info=False, output=False, num_sources=500,
-              headerlet_filenames=None, catalog_list=['GAIAedr3', 'GAIADR2', 'GAIADR1'], fit_label=None,
-              **alignment_pars):
+                  runfile=None, print_fit_parameters=True, print_git_info=False, output=False,
+                  num_sources=500, headerlet_filenames=None, catalog_list=['GAIAedr3', 'GAIADR2', 'GAIADR1'],
+                  fit_label=None, **alignment_pars):
```

Comment:
The parameter files contain a "run_align" section which has a "catalog_list" variable that should be used.  We need to decide on use priority.   What I have done is if a variable is set in a configuration file, I do not provide default values in the method call.  In this way I can distinguish between a user setting the value or reverting to use the values from the parameter file.  FYI - there will be a mosaic_catalog_list which I have in my ticket.

Reply:
This API change will be implemented in a separate PR (yet to come).